src/test/mir-opt/simplify_if.rs

```diff
@@ -1,19 +1,7 @@
+// EMIT_MIR rustc.main.SimplifyBranches-after-const-prop.diff
+
 fn main() {
     if false {
         println!("hello world!");
```

Comment:
we should probably make this test less noisy by doing something trivial here like `std::process::exit(1);`